pylibs/plotting/HistogramPlotter.py

```diff
@@ -48,7 +48,28 @@ def __init__(self, config):
         if not os.path.exists(self.config.output_path):
             os.makedirs(self.config.output_path)
 
+    def histosampleExists(self, hist, sample):
+        for h, s in self.histosamples:
+            if h.getName() == hist.getName() and s.name == sample.name:
+                return True
+        return False
+
+    def histosample2DExists(self, hist, sample):
+        for h, s in self.histosamples2D:
+            if h.getName() == hist.getName() and s.name == sample.name:
+                return True
+        return False
+
+    def histosampleRatioExists(self, hist_pass, hist_tot, sample):
+        for h_pass, h_tot, s in self.ratiosamples:
+            if h_pass.getName() == hist_pass.getName()+'_pass' and h_tot.getName() == hist_tot.getName()+'_tot' and s.name == sample.name:
+                return True
+        return False
```

Comment:
If they are only used inside of HistogramPlotter, they should start with `__` to denote these are private methods.